.pre-commit-config.yaml

```
@@ -18,3 +18,26 @@ repos:
     hooks:
       - id: prettier
         types_or: [css, scss, javascript, ts, tsx, json, yaml]
+  - repo: https://github.com/pre-commit/mirrors-eslint
+    rev: v8.8.0
+    hooks:
+      - id: eslint
+        additional_dependencies:
+          - eslint@7.2.0
+          - eslint-config-airbnb@18.2.1
+          - eslint-plugin-import@2.23.4
+          - eslint-plugin-jsx-a11y@6.4.1
+          - eslint-plugin-react@7.24.0
+          - eslint-plugin-react-hooks@4.3.0
+          - "@babel/core@7.16.7"
+          - "@babel/preset-env@7.14.2"
+          - "@babel/preset-react@7.10.1"
+          - "@wagtail/eslint-config-wagtail@0.4.0"
+          - babel-eslint@10.0.3
```

Installation for Wagtail should be simpler since we use Node 16 / npm v8:

```suggestion
          - eslint@8.8.0
          - "@typescript-eslint/eslint-plugin@5.10.1"
          - "@typescript-eslint/parser@5.10.1"
          - "@wagtail/eslint-config-wagtail@0.4.0"
```